### PR TITLE
Created route api/v1/user/approved_students & created func get_approv…

### LINKED ISSUE #4: create API's for approved students
add API's for retrieving only approved students

### PR DESCRIPTION
**Pull request by dheeraj-israni**
…ed_students in usersDAO.py

# Description

create API's for approved students #4
Added route /approved_students for endpoint api_get_approved_students in users.py
& added get_approved_students func in usersDAO.py

Fixes #4 

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/api/user.py [hunk 1]
```diff
@@ -59,3 +60,12 @@ def login():
 @user_api_v1.route("<id>/create_profile")
 def create_profile():
     pass
+
+@user_api_v1.route("/approved_students")
+def api_get_approved_students():
```
Comment (kmangesh49) [inline app/api/user.py:65]: Change the path to /student/approved

File: app/dao/usersDAO.py [hunk 2]
```diff
@@ -30,3 +30,10 @@ def get_all_companies():
         return list(db["users"].find({"role":"company"},{"password":0}))
     except Exception as e:
         return e
+
+def get_approved_students():
+    try:
+        return list(db["users"].find({"role" : "student", "approved_by_college" : True},
```
Comment (kmangesh49) [inline app/dao/usersDAO.py:36]: Only return fields required as per prototype

Comment (dheeraj-israni) [reply app/dao/usersDAO.py:36]: Okay

Comment (dheeraj-israni) [reply app/dao/usersDAO.py:36]: c1
These fields...?
Database don't have roll number, class & resume fields

Comment (kmangesh49) [reply app/dao/usersDAO.py:36]: above details will be show when requesting for particular id....for approved students only return **_id, name, class, rollno, department**

**Comment by dheeraj-israni** [comment]
I have made the required changes.
When you view the latest changes of commit '36bcaf1' it seems that I have done changes to other code also.... but I have not done changes to the other code, just have pulled the recent changes.

**Pull request merged**